File: Backened/src/Million.Domain/Models/Owner.cs
using System;

namespace Million.Domain.Models
{
    public class Owner
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Address { get; set; } = string.Empty;
        public string Photo { get; set; } = string.Empty;
        public DateTime Birthday { get; set; }
        public string Username { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty; // In a real app, this should be a hashed password.
    }
} 

File: frontend/src/models/owner.js
export class Owner {
    constructor(id, name, address, photo, birthday, username, properties = []) {
        this.id = id
        this.name = name
        this.address = address
        this.photo = photo
        this.birthday = birthday
        this.username = username
        this.properties = properties
    }
}
